src/routes/events.svelte

```diff
@@ -6,35 +6,20 @@
   import EventCarousel from "@/components/events/event-carousel.svelte";
   import CallToActionSection from "@/components/sections/call-to-action-section.svelte";
   import AcmEmpty from "@/components/utils/acm-empty.svelte";
-  import { parseIcalData } from "../lib/parse-ical-data";
-
-  const ICAL_TARGET_URL =
-    "https://calendar.google.com/calendar/ical/738lnit63cr2lhp7jtduvj0c9g%40group.calendar.google.com/public/basic.ics";
 
   let events: AcmEvent[] = [];
 
   onMount(() => {
-    // If the /events route has an event's fragment
-    // attached to the URL, the scroll needs to be
-    // offset slightly upwards.
-    if (location.hash.length > 0) {
-      scrollBy({ top: -200 });
-    }
-
     // Lazily load the ICAL data for the event carousel.
-    fetch(ICAL_TARGET_URL, { mode: "no-cors" })
-      .then((response) => response.text())
+    fetch("../events.json")

```

Comment:
Where is this pointed to?

Reply:
I created a public API endpoint (to act as a proxy server) to see if that _could_ solve the problem. If you go to `acmcsuf.com/events.json`, it will load the array of events.

Reply:
I'm getting a 404.

Would this require a server then?

Reply:
Here is what it looks like based on the preview: <https://acm-csuf-site-cksb0sp2f-ethanthatonekid.vercel.app/events.json>